client-sdks/client-js/src/index.test.ts

```diff
@@ -279,7 +285,7 @@ describe('MastraClient Resources', () => {
     });
 
     it('should stream responses', async () => {
-      const mockChunk = { content: 'test response' };
+      const mockChunk = `0:"test response"\n`;
```

Comment:
The AI SDK Data Stream Protocol expects data stream parts in the format `0:"content"\n` for text content, where:
- 0: is the prefix indicating a text part
- "content" is the content
- \n is the line terminator